modules/n1ql/pages/n1ql-language-reference/let.adoc

```diff
@@ -104,10 +104,9 @@ Find all Sunday flights (`day = 0`) to the Charles De Gaulle airport (`CDG`) on
 [source,N1QL]
 ----
 SELECT t1.airline, t1.destinationairport, sch AS schedule
-FROM `travel-sample` AS t1
-LET sch = ARRAY v FOR v IN t1.schedule WHEN v.day = 0 END <1>
-WHERE type = "route"
-AND t1.destinationairport = "CDG"
+FROM `travel-sample`.inventory.route AS t1
+LET sch = ARRAY v FOR v IN t1.schedule WHEN v.day = 0 END -- <1>
```

Comment:
Ah, I'm guessing the SQL comment markers are for the benefit of people copy/pasting the sql from this source, rather than for Asciidoc itself?

Reply:
Yes. Antora hides the comment markers and makes the callout number uncopyable. The comment markers are there so that the query engine does not trip over the callout number during automated testing.